.github/workflows/release.yml

```diff
@@ -124,16 +127,14 @@ jobs:
           cache-dependency-path: vmware-event-router/go.sum
         id: go
         
-      - id: login_to_gcp
-        uses: google-github-actions/auth@35b0e87d162680511bf346c299f71c9c5c379033
+      - name: Log in to the Container registry
```

Comment:
if you remove the `ENV` in this job, `ko` will default to `ghcr` and automatically log in https://github.com/ko-build/setup-ko/blob/main/action.yml

Reply:
can you please change `imjasonh/setup-ko@ace48d793556083a76f1e3e6068850c1f4a369aa` to `ko-build/setup-ko@v0.6`?